en-US/Punctuation.xml

```diff
@@ -426,19 +426,43 @@
 	 <section id="quotation-marks">
 		<title>Quotation Marks</title>
 
+<!-- Previous guidance
 		 <para>
 			Commas and periods go inside quotation marks.
 		</para>
  		 <para>
-		Question marks, exclamation points, dashes, and semicolons go inside the quotation marks if they are part of the quotation; if not, they go outside. 
-		</para>
-	</section>
+			Question marks, exclamation points, dashes, and semicolons go inside the quotation marks if they are part of the quotation; if not, they go outside. 
+		</para> -->
 
-<!-- Not implementing this change for now.  		
 		 <para>
-			In technical documentation, punctuation marks, including periods, commas, question marks, exclamation points, dashes, and semicolons go inside the quotation marks if they are part of the quotation; if not, they go outside.
+			In technical documentation, punctuation marks, including periods, commas, question marks, exclamation points, dashes, and semicolons go inside the quotation marks if they are part of the quotation or part of a programming element that uses quotation marks; if not, they go outside the quotation marks. If a sentence ends with a quotation, use only one punctuation mark to end the sentence, and place it inside the closing quotation mark if it is part of the quotation, or outside if it is not.
+
 		</para> 
-Add examples of when to use or not to use quotation marks. 
+		<example><title>Correct Examples of the Use of Punctuation with Quotation Marks</title>
+        <para>
+        For bind mounts, add "bind" and set fstype to "none".
+        </para>
+		<screen>
+		Login successful.
+		...output omitted...
+		Using project "default".
+        </screen>
+		<para>
+		As you can see in the previous image, the title of the web page is "OpenShift 3 Etherpad".
+		</para>
+		<para>
+		In the program segment, ensure that the value is X'FF'.
+        </para>		
+		<para>
+		If you use the keyword operand PGMID="PAYCOM", you must ...
```

Comment:
Where did this example come from? I'm not sure how good of an example it is for what we're trying to demonstrate.

Reply:
It came from IBM Style. The idea was to show that the comma goes outside the quotation marks. I'll remove it and we have several other examples anyway.